app/dashboard/templates/dashboard/hackathon/ethhack_2019.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+{% load i18n static %}
+<html lang="en">
+<head>
+  {% include 'shared/head.html' %}
+  <link rel="stylesheet" href="{% static "v2/css/dashboard.css" %}">
+  <link rel="stylesheet" href="{% static "v2/css/ethhack2019.css" %}">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Eth Hackathon 2019</title>
+</head>
+  <body class="text-center" style="background-color: #180944; color: white">
```

Comment:
please take a look at all the utilities classes you have from bootstrap https://getbootstrap.com/docs/4.3/utilities/spacing/

Reply:
fixed